packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -63,3 +69,198 @@ describe("MafsGraph", () => {
         expect(mockChangeHandler).toHaveBeenCalled();
     });
 });
+
+function map(
+    num: number,
+    startMin: number,
+    startMax: number,
+    targetMin: number,
+    targetMax: number,
+): number {
+    return (
+        ((num - startMin) / (startMax - startMin)) * (targetMax - targetMin) +
+        targetMin
+    );
+}
+
+function graphToPixel(
```

Comment:
What do you think of basically copy-pasting the Mafs transform logic into this function (it doesn't look like it's exposed in Mafs anywhere). 

https://github.com/stevenpetryk/mafs/blob/adc9122f102dc18128644d752d07390a27b92a64/src/display/Point.tsx#L21

Reply:
I ended up using Ben's [recommendation](https://github.com/Khan/perseus/pull/1166#discussion_r1559726655) which will use Mafs transform helpers.